src/main/scala/fly/play/aws/AwsCredentials.scala

```diff
@@ -18,12 +18,14 @@ object AwsCredentials extends ((String, String, Option[String]) => AwsCredential
   def apply(accessKeyId: String, secretKey: String, token: Option[String] = None): AwsCredentials =
     SimpleAwsCredentials(accessKeyId, secretKey, token)
 
-  implicit def fromConfiguration(implicit app: Application): AwsCredentials = {
+  implicit def fromConfiguration(implicit app: Application): AwsCredentials = fromConfiguration("aws")
+
+  implicit def fromConfiguration(prefix: String = "aws")(implicit app: Application): AwsCredentials = {
```

Comment:
This one does not need to be implicit. I also think the default value is unneeded.

What do you think about:

```
def fromConfiguration(prefix: String)(implicit app: Application): AwsCredentials
```